source/documentation/deploying_services/index.md

```diff
@@ -1008,6 +1005,102 @@ You must bind your app to the Redis service to be able to access the cache from
 
     If your app writes service connection errors to `STDOUT` or `STDERR`, you can view recent errors with ``cf logs APPNAME --recent``. See the section on [Logs](#logs) for details.
 
+### Connect to a Redis service instance from your local machine
+
+You must use the [Conduit](#conduit) plugin and stunnel tool to connect your local machine to your Redis service instance.
+
+#### Prerequisites
+
+You must:
+
+- install the Redis command line (CLI) tool on your local machine (this is included in the [standard Redis installation](https://redis.io/download) [external link])
+- [log into Cloud Foundry](/#setting-up-the-command-line)
+- [target the space](/#setting-a-target) where your Redis service instance is located
+
+#### Install and configure Conduit
+
+1. Run the following command to install the Conduit plugin:
+
+    ```
+    cf install-plugin conduit
+    ```
+
+1. Start Conduit by running:
+
+    ```
+    $ cf conduit SERVICE_NAME
+    ```
+
+    where `SERVICE_NAME` is a unique descriptive name for this service instance.
+
+    You will see the following output:
+
+    ```
+    service: SERVICE_NAME (redis)
+    host: 127.0.0.1
+    port: 7081
+    username:
+    password: REDIS_PASSWORD
+    db: DATABASE
+    ```
+
+    where `REDIS_PASSWORD` is the password given to you by Conduit.
+
+#### Install and configure stunnel
```

Comment:
A TLS proxy would be a nice feature to add to conduit in the future 🤔

Reply:
Indeed, we talked about this, but decided to document the current approach in the meantime.